.github/workflows/kind-e2e.yaml

```diff
@@ -142,6 +142,9 @@ jobs:
             hostPath: /tmp/etcd
         - role: worker
           image: kindest/node:${{ matrix.k8s-version }}@${{ matrix.kind-image-sha }}
+
+        # networking:
+        #   disableDefaultCNI: true
```

Comment:
Just checking: did you intend to leave this in?

Reply:
I didn't but Kind is actually complaining that it doesn't recognize the "networking" key and breaks the Kind installation, it's odd but I'm trying to find a solution, if you can think of a reason why this happens please let me know

Reply:
What I'm not following is whether we (a) need this config, in which case we shouldn't merge this with it commented out, or (b) we do not need this config, in which case should we just delete it? Could you add a little more detail about what this is for and what happens without it etc?

Reply:
This is supposed to disable kindnet, the default CNI provider that doesn't support NetworkPolicy but when this option is used for some reason it's not recognized by kind however just installing calico seems to work, for some reason I need to add this locally to install calico but it seems to work just fine without it in github actions